event.ts

```diff
@@ -47,3 +47,24 @@ export async function prepareNormalNostrEvent<Kind extends NostrKind = NostrKind
     };
     return sender.signEvent<Kind>(event);
 }
+
+export async function prepareDeletionNostrEvent(
+    sender: NostrAccountContext,
+    ...events: NostrEvent<NostrKind>[]
+): Promise<NostrEvent<NostrKind.DELETE> | Error> {
+    if (events.some((e) => e.pubkey !== sender.publicKey.hex)) {
+        return new Error("deletion events must be from the same account");
```

Comment:
This is not true because any body can attempt to delete, including admins. Other in some relays, maybe there are concepts of moderators for certain kinds of events

Reply:
Okay, there is really no need for this.